client/js/components/user/DpUserListExtendedItem.vue

```diff
@@ -273,16 +288,8 @@ export default {
         }
       }
 
-      // If currently selected department doesn't match current orga, reset to 'Keine Abteilung' or first option of current orga
-      if (typeof this.currentOrganisation.departments.find(dep => dep.id === this.currentDepartment.id) === 'undefined') {
-        this.resetCurrentDepartment()
-      }
-
       return dpApi.patch(url, {}, payload)
-        .then(response => checkResponse(response, {
-          200: { type: 'confirm', text: 'info.user.updated' },
-          204: { type: 'confirm', text: 'info.user.updated' }
-        }))
+        .then(checkResponse)
```

Comment:
Why do you remove those custom notices?

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2673/commits/8a4018547e97324776689b21b7bdf28226bb7a81

Reply:
I fixed the checkResponse and forgot to add the custom notifications. Thanks for adding.